### ISSUE TITLE
GPL-824 Include Dockerfile inside Limber repo as a tool to use for local development (C=S, V=3*)

### ISSUE THREAD

**Issue by emrojo** [sanger/limber#605]
**User story**
As a developer I would like to have a Dockerfile in the project that I could use to run a local environment of the app in case the machine we are using for development have some misconfiguration of libraries. This way we can speed up coding time when we have some problems on libraries installed in local

**Who are the primary contacts for this story**
Eduardo Martin

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Build a docker image from the dockerfile
- [ ] Run a set of tests using that docker image

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
Duplicate. Completed as another story.
